apps/web/src/assets/css/filter-overrides.scss

```
@@ -0,0 +1,133 @@
+// Custom overrides on top of the MOJ Frontend filter component + filter-layout
+// object (adopted in #892). These bring the migrated panels back towards the
+// bespoke look the pages had before the migration, in the same spirit as
+// hmcts/pip-frontend's main.scss filter overrides. Loaded from web.scss AFTER
+// the MOJ @use lines and govuk index so these rules win on source order.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)